### DISCUSSION TITLE
Corsica data available

### DISCUSSION THREAD

**Discussion by dos934** [electricitymaps/electricitymaps-contrib#3964] [Ideas] [answered]
Hi everyone,

Since 2 days, Corsica real time energy mix is available : https://opendata-corse.edf.fr/pages/page-prod_electricite_filiere_cout/

 I think it is not difficult to put in in electricity map since it is the same code as réunion opendata site : https://opendata-reunion.edf.fr/pages/page-prod_electricite_filiere_cout0/ which is already in electricity map 

Someone available to do it ?:)

**Comment by VIKTORVAV99** [comment, marked as answer]
I have created PR #3978 that should add the live data to the map once it's merged and a new version is deployed.
